packages/grid/_modules_/grid/components/containers/GridMainContainer.tsx

```diff
@@ -0,0 +1,5 @@
+import * as React from 'react';
+
+export function GridMainContainer(props: React.PropsWithChildren<{}>) {
+  return <div className="MuiDataGrid-mainGridContainer">{props.children}</div>;
```

Comment:
We have in the `unstyled` package util for generating classes, that can be used like this:

```
const badgeUnstyledClasses = generateUtilityClasses('MuiBadge', [
  'root',
  'badge',
  'dot',
  'invisible',
]);
```

It helps keep consistency between the classes, plus access to the classes as constants where necessary. I am mentioning this because I was expecting that the className for the component `GridMainContainer` would be `MuiDataGrid-mainContainer` or at least `MuiDataGrid-gridMainContainer`, but it is different.

Reply:
If you do not want dependency to the unstyled package, at least this util can be duplicated, if you think it will bring some value...

Reply:
As discussed, it's a bit early stage to integrate the unstyled package in X
I will rename the class :)